packages/react-urql/examples/1-getting-started/public/sw.js

```diff
@@ -0,0 +1,50 @@
+/* NOTE:
+ * This demo is running a mock API in a service worker!
+ * In this file you can find the service worker. It doesn't
+ * execute GraphQL requests itself. Rather, it'll send them
+ * back to the client-side, which creates the response.
+ */
+
+const channel = new BroadcastChannel("cs-graphql");
+
+let count = 0;
+
+self.addEventListener("install", () => {
+  self.skipWaiting();
+});
+
+self.addEventListener("activate", event => {
+  event.waitUntil(self.clients.claim());
+});
+
+self.addEventListener("fetch", async event => {
+  const { request } = event;
+  if (!/\/graphql$/.test(request.url) || request.url.includes("node_modules")) {
```

Comment:
This was a slight gotcha. Turns out if we resolve the GraphQL package there’s a chance the version isn’t included, so an extra check is needed here